xbmc/pvr/addons/PVRClients.cpp

```diff
@@ -258,6 +258,40 @@ bool CPVRClients::GetClientName(int iClientId, std::string &strName) const
   return bReturn;
 }
 
+std::vector<SBackend> CPVRClients::GetBackendProperties() const
+{
+  std::vector<SBackend> backendProperties;
+  CSingleLock lock(m_critSection);
+
+  for (const auto &entry : m_clientMap)
+  {
+    const auto &client = entry.second;
+    
+    if (!client->ReadyToUse())
+      continue;
+
+    SBackend properties;
```

Comment:
Basically because SBackendProperties would be one object, but what would you call a vector of those? You can't write `properties'` like in standard English.